.github/workflows/learninh-github-actions.yml

```diff
@@ -0,0 +1,13 @@
+name: Github Actions test
+run-name: ${{ github.actor }} is leaarning GitHub Actions 🚀
+on: [push]
```

Comment:
Push and pull request

Reply:
i changed this
now the trigger it'll call it on push and pull_request